src/index.html

```diff
@@ -4,9 +4,390 @@
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- HEADER -->
+    <header class="header">
+      <a href="#image" class="header__logo">
+        <img
+          class="header__image"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <!-- NAV -->
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link nav__link--active is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Smartphones" class="nav__link">Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- MAIN -->
+    <main class="main">
+
+      <div
+      class="product-card"
+      data-qa="card"
+      >
+
+        <img
+          class="product-card__image"
+          src="images\imac.jpeg"
+          alt="APPLE A1419 iMac"
+        >
+
+        <h2 class="product-card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="product-card__product-code">Product code: 195434</p>
+
+        <div class="product-card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="product-card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="product-card__price">
+          <p class="product-card__text">Price:</p>
+          <p class="product-card__amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="product-card__button"
+          data-qa="card-hover"
+        >
+          <span class="product-card__button-text">Buy</span>
```

Comment:
This tag span is redundant here, so you need to remove it everywhere
```suggestion
          Buy
```